src/components/PoolInfoCard.tsx

```diff
@@ -204,6 +208,27 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
             </Box>
           ))}
         </Box>
+        <Typography mt={3}>
+          Pool address:&nbsp;
```

Comment:
instead of using &nbps; can you just appropriate padding/margin to the component?